install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: meshery-nighthawk
+description: Meshery Adapter for App Mesh chart.
+icon: https://meshery.io/images/logos/meshery-logo.png
```

Comment:
We have the meshery-nighthawk icon: https://github.com/meshery/meshery-nighthawk/blob/master/assets/img/icon/nighthawk-logo.png, can we use that? // @leecalcote